src/ws/service/friendship_event_updates.rs

```diff
@@ -48,6 +48,18 @@ pub async fn handle_friendship_update(
     )
     .await?;
 
+    let room_message_body = event_payload.request_event_message_body.as_deref();
+
+    // The room may exists but maybe the current user hasn't joined it yet.
+    accept_room_invitation(
```

Comment:
Let's also update the doc for the `accept_room_invitation` function as this should be done either way (when accepting the invite or not).